lib/chef/file_access_control/windows.rb

```diff
@@ -255,10 +255,7 @@ def calculate_flags(rights)
             flags |= CONTAINER_INHERIT_ACE
           when :objects_only
             flags |= OBJECT_INHERIT_ACE
-          when true
-            flags |= CONTAINER_INHERIT_ACE
-            flags |= OBJECT_INHERIT_ACE
-          when nil
+          when true, nil
```

Comment:
Maybe just `else` here?

Reply:
I'm not sure if there are additional values we wouldn't want to perform the action for though. Else might be fine, but it could also change the behavior.

Reply:
False for example would get caught by the else